### PR TITLE
[JENKINS-65161] remove commons-digester2 from core and upgrade plugins to commons-digester3

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

**Comment by olamy** [comment]
see https://github.com/jenkinsci/jenkins/pull/5320

**Comment by olamy** [comment]
@mig42 ping as you are the main maintainer

**Comment by mig42** [comment]
Sure, we'll get this merged and released ASAP 😊

File: pom.xml [hunk 1]
```diff
@@ -51,6 +51,17 @@
         </developer>
     </developers>
 
+    <dependencyManagement>
+      <dependencies>
+        <dependency>
+          <groupId>commons-beanutils</groupId>
+          <artifactId>commons-beanutils</artifactId>
+          <!-- 1.9.4 when this merged https://github.com/stapler/stapler/pull/211 -->
```
Comment (rsandell) [inline pom.xml]: It is merged now

**Review by rsandell — DISMISSED** [review]

File: pom.xml (hunk 1 above)
Comment (olamy) [reply pom.xml]: fixed thanks!

File: src/main/java/com/codicesoftware/plasticscm/plugins/mergebot/jenkins/PlasticChangeLogParser.java [hunk 2]
```diff
@@ -1,58 +1,73 @@
...
-        return new PlasticChangeLogSet(run, browser, changesetList);
-    }
-}
+package com.codicesoftware.plasticscm.plugins.mergebot.jenkins;
+
+import hudson.model.Run;
+import hudson.scm.ChangeLogParser;
+import hudson.scm.ChangeLogSet;
+import hudson.scm.RepositoryBrowser;
+import org.apache.commons.digester3.Digester;
```
Comment (MRamonLeon) [inline src/main/java/com/codicesoftware/plasticscm/plugins/mergebot/jenkins/PlasticChangeLogParser.java:7]: The important line

**Review by MRamonLeon — APPROVED** [review]

**Review by alecharp — APPROVED** [review]

**Comment by alecharp** [comment]
@mig42 any blockers to get this PR merge and cut a release of the plugin? Thanks.

**Comment by olamy** [comment]
@mig42 ping :)

**Review by mig42 — APPROVED** [review]
Looks good to me! I'll merge the PR once I verified it works in my environment.

**Pull request merged**